### PR TITLE
DT-825 - set up internationalization on workflows page

### PR DESCRIPTION
**Pull request by rossedfort**
## Description & motivation 💭 
This PR adds (English only) translation strings to the Recent Workflows page. Note - it does not include holocene components.

### Screenshots (if applicable) 📸 
N/a
### Design Considerations 🎨 
N/a
### How was this tested 👻 

- [x] Manual testing
- [ ] E2E tests added
- [x] Unit tests added

### CODE REVIEW

File: src/lib/i18n/locales/en/workflows.ts [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    'If you supply a custom reason, "{{placeholder}}" will be appended to it.',
+  'batch-terminate-all-success':
+    'The batch $t(terminate) request is processing in the background.',
+  'batch-cancel-all-success':
+    'The batch $t(cancel) request is processing in the background.',
+  'batch-terminate-success': 'Successfully $t(terminated) {{count}} workflows.',
+  'batch-cancel-success': 'Successfully $(canceled) {{count}} workflows.',
+  'configure-workflows': 'Configure Workflow List',
+  'configure-workflows-description':
+    'Add (<1></1>), re-arrange (<2></2>), and remove (<3></3>), Workflow Headings to personalize the Workflow List Table.',
```
Comment (rossedfort) [inline src/lib/i18n/locales/en/workflows.ts:36]: I need to revisit how to translate copy with markup in it, i.e. `This is some <strong>copy</strong>`

File: server/server/route/ui.go [hunk 2]
```diff
@@ -45,6 +45,7 @@ func SetUIRoutes(e *echo.Echo, publicPath string, assets fs.FS) error {
 	e.GET("/logo*", assetsHandler)
 	e.GET("/Temporal_Logo_Animation.gif", assetsHandler)
 	e.GET("/site.webmanifest", assetsHandler)
+	e.GET("/i18n/*", assetsHandler)
```
Comment (robholland) [inline server/server/route/ui.go:48]: I can't comment on most of the PR, but this looks fine.

File: pnpm-lock.yaml [hunk 3]
```diff
@@ -1,372 +1,272 @@
-lockfileVersion: '6.0'
+lockfileVersion: 5.4
```
Comment (laurakwhit) [inline pnpm-lock.yaml:1]: Looks like you used a newer version of `pnpm` here 📜   We should all update so our `pnpm-lock` files will look the same when generating.

File: src/lib/components/workflow-status.svelte [hunk 4]
```diff
@@ -1,19 +1,20 @@
...
-    Running: 'Running',
-    TimedOut: 'Timed Out',
-    Completed: 'Completed',
-    Failed: 'Failed',
-    ContinuedAsNew: "Cont'd as New",
-    Canceled: 'Canceled',
-    Terminated: 'Terminated',
-    Paused: 'Paused',
+    Running: translate('workflows', 'running'),
+    TimedOut: translate('workflows', 'timed-out'),
```
Comment (laurakwhit) [inline src/lib/components/workflow-status.svelte:11]: Looks like something isn't working with `Timed Out`

Screenshot 2023-05-26 at 1 18 52 PM

File: package.json [hunk 5]
```diff
@@ -33,6 +33,7 @@
     "dev:docker": ". ./.env && VITE_API=http://localhost:8080 vite dev --port 3000",
     "dev:cloud": "VITE_TEMPORAL_UI_BUILD_TARGET=cloud vite dev --port 3000",
     "dev:ui-server": "vite dev --mode ui-server --port 3000",
+    "dev:ui-only": "vite dev --mode ui-only --port 3000",
```
Comment (laurakwhit) [inline package.json:36]: I'm getting an
 ```js 
Error: Not found: /i18n/locales/fr/workflows.json
```
 error when running this.

File: src/lib/pages/workflows-with-new-search.svelte [hunk 6]
```diff
@@ -242,7 +237,9 @@
 
 <header class="mb-2 flex justify-between">
   <div>
-    <h1 class="text-2xl" data-cy="namespace-title">Recent Workflows</h1>
+    <h1 class="text-2xl" data-cy="namespace-title">
```
Comment (laurakwhit) [inline src/lib/pages/workflows-with-new-search.svelte:240]:
```suggestion
    <h1 class="text-2xl" data-cy="workflows-title">
```

File: src/lib/components/workflow/advanced-filter/status-filter.svelte [hunk 7]
```diff
@@ -3,19 +3,20 @@
...
   const statuses = {
-    All: 'All',
-    Running: 'Running',
-    'Timed Out': 'TimedOut',
-    Completed: 'Completed',
-    Failed: 'Failed',
-    'Continued as New': 'ContinuedAsNew',
-    Canceled: 'Canceled',
-    Terminated: 'Terminated',
+    All: translate('all'),
```
Comment (laurakwhit) [inline src/lib/components/workflow/advanced-filter/status-filter.svelte:12]: Do we need `translate` here? We are already translating `humanFriendlyNames` in `src/lib/components/workflow-status.svelte`.

File: src/lib/components/workflow/dropdown-filter/workflow-status.svelte [hunk 8]
```diff
@@ -7,21 +7,24 @@
...
-  const AllStatuses = {
-    All: 'All',
-    Running: 'Running',
-    'Timed Out': 'TimedOut',
-    Completed: 'Completed',
-    Failed: 'Failed',
-    'Continued as New': 'ContinuedAsNew',
-    Canceled: 'Canceled',
-    Terminated: 'Terminated',
+  export const AllStatuses = {
```
Comment (laurakwhit) [inline src/lib/components/workflow/dropdown-filter/workflow-status.svelte:13]: Same here.

File: pnpm-lock.yaml (hunk 3 above)
Comment (rossedfort) [reply pnpm-lock.yaml:1]: Ah interesting, I didn't do anything to upgrade, must have been some auto-update. I'll revert.

File: src/lib/components/workflow/advanced-filter/status-filter.svelte (hunk 7 above)
Comment (rossedfort) [reply src/lib/components/workflow/advanced-filter/status-filter.svelte]: Good catch, I didn't realize that neither the keys or values of this object are actually displayed in the DOM

File: src/lib/models/workflow-status.ts [hunk 9]
```diff
@@ -0,0 +1,19 @@
...
+  'Failed',
+  'ContinuedAsNew',
+  'Canceled',
+  'Terminated',
+] as const;
+
+export const workflowStatusFilters: WorkflowFilters = [
+  ...workflowStatuses,
+  'All',
+] as const;
```
Comment (laurakwhit) [inline src/lib/models/workflow-status.ts:19]: Looks like `All` got moved to the end of the list ⬇️ 
Screenshot 2023-05-30 at 3 09 30 PM
and we probably still want it at the top

```suggestion
export const workflowStatusFilters: WorkflowFilters = [
  'All',
  ...workflowStatuses,
] as const;
```

**Review by laurakwhit — APPROVED** [review]
We'll want to update (or resolve) that `pnpm-lock.yaml` file before merging 🚀

**Pull request merged**